CSS/main.css

```diff
@@ -72,18 +85,31 @@ ul {
 
 .main {
     grid-area: main;
```

Comment:
I personally avoid using keywords as the names for classes or IDs. For example, main and footer are elements/tags and here they are class-names also. 
What I do is add another word with the tag-name, for example, main-container or anything, so that it is distinctive.